nss-day-cohort-41/tabloid-fullstack-first-amendment-dancers-andy
# Ba posts

# Description
The user can view a list of all approved posts sorted by date, a list of all of their own posts, a post details page, and can add a post.
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
# Testing Instructions
Checkout this branch.  Ensure that all previously working features are still functional.  Verify that you can add a post as a specific user.  Verify that you can view post lists and details.
# Checklist:
